modules/claims_api/app/models/claims_api/intent_to_file.rb

```diff
@@ -2,12 +2,29 @@
 
 module ClaimsApi
   class IntentToFile
+    attr_reader :id, :creation_date, :expiration_date, :status, :type
+
     ITF_TYPES = {
       'compensation' => 'C',
       'burial' => 'S',
-      'pension' => 'P'
+      'pension' => 'P',
+      'C' => 'compensation',
```

Comment:
why add the ITF types to go both ways? wouldn't we want to just have a one-way conversion to enforce less values?

Reply:
ie, having be either
```ruby
ITF_TYPES = {
 'compensation' => 'C',
 'burial' => 'S',
 'pension' => 'P'
}
```
or 
```ruby
ITF_TYPES = {
 'C' => 'compensation',
 'S' => 'burial',
 'P' => 'pension'
}
```

Reply:
Eh, I mainly just did this to support line#23 of the initializer.
(`@type = ITF_TYPES[type]`)

It just made sense to me to go ahead and add the ability to map the other direction in that constant, rather than adding logic in the initializer to do that conversion.  But I'm down for alternative ideas if we don't like this.

Reply:
Oh I think I see now. BGS provides that code in letters and we want to provide it in full strings. I would maybe just refactor it into separate hashes to make the division more explicit. E.g. keep renaming the existing hash to `ITF_TYPES_TO_BGS` and then the new one `BGS_TO_ITF_TYPES`. or something similar. any feelings about that modification?

Reply:
I'm totally down for that.  I was trying to avoid having inverse constants, but if it introduces confusion, then yeah I'm totally cool with having 2 separate mappings.  I'll make that change and update the PR.

Reply:
yeah, I think as long as they're appropriately named constants, inverses of each other is easier to grok than having everything in the same hash

Reply:
That change has been made [here](https://github.com/department-of-veterans-affairs/vets-api/pull/7998/commits/b8927734b5ad34373b0bae5894d51a653a7cd15c).